ci.sh

```diff
@@ -42,6 +42,10 @@ FEATURES_CHECK=(
     "defmt,alloc,medium-ip,medium-ethernet,proto-ipv6,proto-ipv6,proto-igmp,proto-dhcpv4,socket-raw,socket-udp,socket-tcp,socket-icmp,socket-dns,async"
 )
 
+FEATURES_NO_STD=(
+    "alloc,defmt,medium-ethernet,medium-ip,medium-ieee802154,proto-ipv4,proto-ipv4-fragmentation,proto-igmp,proto-dhcpv4,proto-ipv6,proto-rpl,proto-sixlowpan,proto-sixlowpan-fragmentation,proto-dns,socket,socket-raw,socket-udp,socket-tcp,socket-icmp,socket-dhcpv4,socket-dns,socket-mdns,packetmeta-id,async"
+)
+
```

Comment:
Can you remove this? I don't think you are using it. Or you could add it to `FEATURES_CHECK`, which already contains some `no_std` tests.

Reply:
You are indeed right. I wanted to use it but then copy and pasted `FEATURES_CHECK`. The intention of `FEATURES_NO_STD` was to only build once with all features that work with no-std. But we can also use the existing  `FEATURES_CHECK`.